speculation-rules.bs

```diff
@@ -629,6 +639,97 @@ A <dfn>prerender candidate</dfn> is a [=struct=] with the following [=struct/ite
   1. [=Assert=]: This step is not reached.
 </div>
 
+<h2 id="content-security-policy">Content Security Policy</h2>
+Inline script use is restricted by [[CSP]], but there are cases where a developer doesn't want to permit general inline scripts, but wants to permit inline speculation rules. So, we make the following change that extends [[CSP]] to support a new source keyword for inline speculation rules.
+Also, this changes affect 
```

Comment:
Nit: This sentence trails off...

Reply:
oh, thanks. this line was mistakenly added in the rev d98e...